### ISSUE TITLE
gantt can't edit  when reinit

### ISSUE THREAD

**Issue by dzchengang** [DHTMLX/gantt#29]
![1](https://user-images.githubusercontent.com/9650687/45146812-25db4b80-b1f6-11e8-9b95-3ebb1bf7bbd7.png)
![2](https://user-images.githubusercontent.com/9650687/45146815-25db4b80-b1f6-11e8-80fa-608d0c3cbbd8.png)

gantt  can't edit when reinit

```
 gantt.config.columns =[
					{
						label: 'name',
						name: "text",
						tree: true,
						resize: true,
						width: 250,
						editor: {
							type: "text",
							map_to: "text"
						}
					},
					
				];
    	
    gantt.init("gantt_here");
	gantt.parse(tasks);
```
				
	function reinit() {
		gantt.init("gantt_here2");
	}
</script>
</body>'

**Comment by ouassimBenMosbah** [comment]
I encountered the same issue ! Do you know if there is any solution ?
thanks

**Comment by dzchengang** [comment]
edit dthmlxgantt.js    
![image](https://user-images.githubusercontent.com/9650687/46523998-0cdbbe00-c8ba-11e8-85ac-0b8432cd7a96.png)
the reasion is trigger event repeat

**Comment by AlexKlimenkov** [comment]
Hi,
should be fixed in [6.0](https://github.com/DHTMLX/gantt/releases/tag/6.0.0)
